src/components/AdminSettings.vue

```diff
@@ -231,11 +374,19 @@ export default {
 				server: F_MODES.EDIT,
 				opOauth: F_MODES.DISABLE,
 				ncOauth: F_MODES.DISABLE,
+				opUserAppPassword: F_MODES.DISABLE,
+				projectFolderSetUp: F_MODES.DISABLE,
 			},
 			isFormCompleted: {
-				server: false, opOauth: false, ncOauth: false,
+				server: false, opOauth: false, ncOauth: false, opUserAppPassword: false, projectFolderSetUp: false,
+			},
+			buttonTextLabel: {
+				keepCurrentChange: t('integration_openproject', 'Keep current change'),
```

Comment:
The wording has changed to, but this can be done in next PR just mentioning here as a reminder 
```suggestion
				keepCurrentChange: t('integration_openproject', 'Keep current setup'),
```